compose.testing.yaml

```
@@ -8,7 +8,8 @@ services:
     # https://docs.docker.com/reference/compose-file/build/
     build:
       # https://docs.docker.com/reference/compose-file/build/#target
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)